8cee7c4a-8047-4cb9-a86d-f7fd87b2d1f8.json

```diff
@@ -0,0 +1,8 @@
+{
+  "sourceDefinitionId": "8cee7c4a-8047-4cb9-a86d-f7fd87b2d1f8",
+  "name": "DataScope",
+  "dockerRepository": "airbyte/datascope",
+  "dockerImageTag": "0.1.0",
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/datascope",
+  "icon": "datascope.svg"
```

Comment:
there is currently a bug that breaks old versions of airbyte when adding icons for new connectors. We'll fix this soon but for now let's just remove that field (but keep the SVG in the repo for future use)
```suggestion
```